<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guestApplications</class>
 <widget class="QDialog" name="guestApplications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Оставьте заявку</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFIO">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите ФИО...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePhone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите телефон...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEmail">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите email...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheck">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teResult">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Оставьте Ваши контакты, чтобы мы могли связаться с Вами:</string>
   </property>
  </widget>
  <zorder>btnBack</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>leFIO</zorder>
  <zorder>lePhone</zorder>
  <zorder>leEmail</zorder>
  <zorder>btnCheck</zorder>
  <zorder>label_4</zorder>
  <zorder>teResult</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
